app/controllers/admins/messages_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Admins
+  class MessagesController < BaseController
+    
+  def index
+    @message = Message.all.order created_at: :desc
```

Comment:
remove `all` and wrap parameters to parentheses

Reply:
what does it mean?